.clang-format

```diff
@@ -11,6 +11,7 @@ ColumnLimit: 100
 CompactNamespaces: false
 IndentWidth: 4
 NamespaceIndentation: None
+DerivePointerAlignment: false
```

Comment:
https://stackoverflow.com/questions/56537847/why-do-the-pointeralignment-options-not-work